repose-aggregator/tests/performance-tests/roles/repose/files/config/services/uriredaction/uri-redaction.cfg.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<uri-redaction xmlns="http://docs.openrepose.org/repose/uri-redaction-service/v1.0">
+    <!-- Redact the third path segment of a four segment URI hitting the v1 API. -->
+    <redact>^/v1/[^/]+/([^/]+)/[^/]+.*</redact>
+    <!-- Redact the third path segment of a four segment URI hitting the v2 API's admin endpoint. -->
+    <redact>^/v2/admin/([^/]+)/[^/]+.*</redact>
```

Comment:
I know Mario wanted a bunch of regexes here to really gauge the effect of applying all regexes. I would actually love to see a config where multiple redactions apply to each request, and multiple redactions do not.

Reply:
Unless we run set of redactions as their own scenario, then they would all be intermixed in the run and the numbers would meld together. So do we want multiple scenarios? Multiple Simulations?

Reply:
Right. Personally, I'm fine with just one number right now. In my mind, we just want to capture something close to a worst-case scenario so that we can say with reasonable certainty that the service will be at least so fast in most use-cases.

Although, I'm also not against the idea of running multiple scenarios... but I certainly wouldn't require it for this story.